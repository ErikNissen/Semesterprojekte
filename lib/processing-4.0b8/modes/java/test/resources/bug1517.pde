version https://git-lfs.github.com/spec/v1
oid sha256:48473619fd35508ae5ae8c99d1f68ca91751bb6a432d8c0a91421f39a720a9ee
size 421
