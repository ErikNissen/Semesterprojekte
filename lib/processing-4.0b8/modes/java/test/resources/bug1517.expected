version https://git-lfs.github.com/spec/v1
oid sha256:532fe543f8e833353f735539f98a2ba456f550797dad71e464f392428ac4e34b
size 1077
